Index: resources/lang/zh-CN/device-category.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?php\r\nreturn [\r\n    'labels' => [\r\n        'DeviceCategory' => '设备分类',\r\n    ],\r\n    'fields' => [\r\n        'name' => '名称',\r\n        'description' => '描述',\r\n    ],\r\n    'options' => [\r\n    ],\r\n];\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- resources/lang/zh-CN/device-category.php	(revision 1fd1d73d71ed6ce9cd788f5552e90ec8b7db54d7)
+++ resources/lang/zh-CN/device-category.php	(date 1603162157353)
@@ -2,6 +2,7 @@
 return [
     'labels' => [
         'DeviceCategory' => '设备分类',
+        'categories' => '设备分类'
     ],
     'fields' => [
         'name' => '名称',
Index: resources/lang/zh-CN/device-track.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?php\r\nreturn [\r\n    'labels' => [\r\n        'DeviceTrack' => '设备归属记录',\r\n    ],\r\n    'fields' => [\r\n        'device' => [\r\n            'name' => '设备'\r\n        ],\r\n        'staff' => [\r\n            'name' => '雇员'\r\n        ],\r\n    ],\r\n    'options' => [\r\n    ],\r\n];\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- resources/lang/zh-CN/device-track.php	(revision 1fd1d73d71ed6ce9cd788f5552e90ec8b7db54d7)
+++ resources/lang/zh-CN/device-track.php	(date 1603162157367)
@@ -2,6 +2,7 @@
 return [
     'labels' => [
         'DeviceTrack' => '设备归属记录',
+        'tracks' => '设备归属记录'
     ],
     'fields' => [
         'device' => [
Index: resources/lang/zh-CN/device-record.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?php\r\nreturn [\r\n    'labels' => [\r\n        'DeviceRecord' => '设备',\r\n        'Category' => '分类',\r\n        'Vendor' => '制造商'\r\n    ],\r\n    'fields' => [\r\n        'qrcode' => '二维码',\r\n        'name' => '名称',\r\n        'description' => '描述',\r\n        'category' => [\r\n            'name' => '分类'\r\n        ],\r\n        'vendor' => [\r\n            'name' => '制造商'\r\n        ],\r\n        'sn' => '序列号',\r\n        'mac' => 'MAC',\r\n        'ip' => 'IP',\r\n        'photo' => '照片',\r\n        'owner' => '雇员',\r\n        'department' => '部门'\r\n    ],\r\n    'options' => [\r\n    ],\r\n];\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- resources/lang/zh-CN/device-record.php	(revision 1fd1d73d71ed6ce9cd788f5552e90ec8b7db54d7)
+++ resources/lang/zh-CN/device-record.php	(date 1603162157360)
@@ -3,7 +3,8 @@
     'labels' => [
         'DeviceRecord' => '设备',
         'Category' => '分类',
-        'Vendor' => '制造商'
+        'Vendor' => '制造商',
+        'records' => '设备'
     ],
     'fields' => [
         'qrcode' => '二维码',
Index: resources/lang/zh-CN/software-record.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?php\r\nreturn [\r\n    'labels' => [\r\n        'SoftwareRecord' => '软件',\r\n        'Category' => '分类',\r\n        'Vendor' => '制造商',\r\n        'Left Counts' => '剩余授权数量'\r\n    ],\r\n    'fields' => [\r\n        'qrcode' => '二维码',\r\n        'name' => '名称',\r\n        'description' => '描述',\r\n        'category' => [\r\n            'name' => '软件分类'\r\n        ],\r\n        'version' => '版本',\r\n        'vendor' => [\r\n            'name' => '制造商'\r\n        ],\r\n        'price' => '价格',\r\n        'purchased' => '购入时间',\r\n        'expired' => '过保时间',\r\n        'distribution' => '发行方式',\r\n        'sn' => '序列号',\r\n        'counts' => '授权数量'\r\n    ],\r\n    'options' => [\r\n    ],\r\n];\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- resources/lang/zh-CN/software-record.php	(revision 1fd1d73d71ed6ce9cd788f5552e90ec8b7db54d7)
+++ resources/lang/zh-CN/software-record.php	(date 1603162157373)
@@ -4,7 +4,8 @@
         'SoftwareRecord' => '软件',
         'Category' => '分类',
         'Vendor' => '制造商',
-        'Left Counts' => '剩余授权数量'
+        'Left Counts' => '剩余授权数量',
+        ''
     ],
     'fields' => [
         'qrcode' => '二维码',
